### PR TITLE
Improve import speed

### PR DESCRIPTION
**Pull request by aghaynes**
This seems to cut the import time by ca 60% (on a 3MB export without audit trail). The bottlenecks were in date conversion and factorization. Factorize is still the bottleneck, but I'm not sure how to speed it up further. The one idea I had broke things big time....

### CODE REVIEW

**Comment by aghaynes** [comment]
this is related to #204

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# secuTrialR 1.0.4
+* improved import speed, specifically through changes in dates_secutrial and factorize_secutrial
```
Comment (PatrickRWright) [inline NEWS.md:2]: can you tag the # github issue?

**Comment by PatrickRWright** [comment]
Should we add `readr` reading in this PR?

e.g.
[Here](https://github.com/SwissClinicalTrialOrganisation/secuTrialR/blob/master/R/read_export_table.R#L87) and [here](https://github.com/SwissClinicalTrialOrganisation/secuTrialR/blob/master/R/read_export_table.R#L94)

**Comment by aghaynes** [comment]
> I have spent about 40 minutes looking into this option and, for now, suggest not doing this. At least not right now... The way `readr` reads data seems to break a lot of tests and I don't consider this update important enough to put the refactoring effort in right now.

Doesn't surprise me... from what I remember, readr does all sorts of interpretation stuff (dates?).... maybe `vroom` does less...?

**Comment by aghaynes** [comment]
FWIW... lots of people seem to be having the failed builds due to mac devel - [https://github.com/r-lib/actions/issues/140](https://github.com/r-lib/actions/issues/140)

**Comment by PatrickRWright** [comment]
> > I have spent about 40 minutes looking into this option and, for now, suggest not doing this. At least not right now... The way `readr` reads data seems to break a lot of tests and I don't consider this update important enough to put the refactoring effort in right now.
> 
> Doesn't surprise me... from what I remember, readr does all sorts of interpretation stuff (dates?).... maybe `vroom` does less...?

Lets look into it in a later PR.

Are you done here? If yes, then I'll approve and merge. Thanks for looking into the macOS fail. I suspected something of the sort.

**Comment by aghaynes** [comment]
yes, I think I'm done, thanks

**Pull request merged**